### PR TITLE
OD-90 [Feature] Users will be able to see amount of the selected filters

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 

## Issue
https://weboo.atlassian.net/browse/OD-90

## Description
We have added a function that counts the amount of the selected filters.

## Screenshots/screencasts
https://share.getcloudapp.com/ApuG0mbl

## Backward compatibility
This change is fully backward compatible.

## Reviewers
@AndrRyaz @ivandevupp

### CODE REVIEW

File: js/utils.js [hunk 1]
```diff
@@ -1625,7 +1647,8 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
     },
     Page: {
       updateSearchContext: updateSearchContext,
-      updateFilterControlsContext: updateFilterControlsContext
+      updateFilterControlsContext: updateFilterControlsContext,
+      showSelectedFiltersAmount: showSelectedFiltersAmount
```
Comment (tonytlwu) [inline js/utils.js:1651]:
```suggestion
      updateFilterSelectionCount: updateFilterSelectionCount
```

File: js/utils.js [hunk 2]
```diff
@@ -262,6 +262,28 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
...
+   * This function is used to show amount of the selected by users filters
+   *
+   * @param {Object} options - incoming object with keys:
+   *  filtersInOverlay { Boolean } - represent us if filters shown in the overlay
+   *  $target { Jquery instance } - Jq instance on which user have pressed
+   *
+   * @return {void} this funtion doesn't return anything it add changes directly to the DOM
+   */
+  function showSelectedFiltersAmount(options) {
+    if (!options.filtersInOverlay || !options.$target.length) {
```
Comment (tonytlwu) [inline js/utils.js:275]:
```suggestion
    if (!options.filtersInOverlay || !options.$target || !options.$target.length) {
```

File: templates/build/agenda-filters.build.hbs [hunk 3]
```diff
@@ -7,6 +7,7 @@
             <span class="panel-title-text">
               \{{[name]}}
             </span>
+            <span class="panel-title-amount"></span>
```
Comment (tonytlwu) [inline templates/build/agenda-filters.build.hbs:10]:
```suggestion
            <span class="filter-selection-count"></span>
```

File: js/utils.js [hunk 4]
```diff
@@ -262,6 +262,28 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
...
+      return;
+    }
+
+    var $filterPanel = options.$target.parents('.panel');
+    var activeFiltersAmount = $filterPanel.find('[data-filter-group] .mixitup-control-active').length;
```
Comment (tonytlwu) [inline js/utils.js:280]:
```suggestion
    var activeFiltersSelected = $filterPanel.find('[data-filter-group] .mixitup-control-active').length;
```

File: js/utils.js [hunk 5]
```diff
@@ -262,6 +262,28 @@ Fliplet.Registry.set('dynamicListUtils', (function() {
...
+    var $filtersAmount = $filterPanel.find('.panel-heading .panel-title .panel-title-amount');
```
Comment (tonytlwu) [inline js/utils.js:281]:
```suggestion
    var $count = $filterPanel.find('.panel-heading .panel-title .panel-title-amount');
```

**Review by tonytlwu — CHANGES_REQUESTED** [review]

**Comment by tonytlwu** [comment]
@YaroslavOvdii I updated my suggestions to use the phrase "active filter" to match the terminologies used in https://github.com/Fliplet/fliplet-widget-dynamic-lists/pull/488

**Comment by YaroslavOvdii** [comment]
@tonytlwu Added suggestions

**Review by tonytlwu — APPROVED** [review]
@sofiiakvasnevska This is ready for testing

Risk: High
Affected: LFD users with filters enabled
Verify: LFD filters work correctly

Squash and merge: Add active filter count to filter overlays

**Pull request merged**